sphinx/source/docs/user_guide/graph.rst

```diff
@@ -44,111 +40,157 @@ Here's a code snippet that:
     from bokeh.models import GraphRenderer, Ellipse
     from bokeh.palettes import Spectral8
 
+* List the nodes and initialize a plot.
+
+.. code-block:: python
+
```

Comment:
@Konntroll these code snippets currently look like this:

<img width="604" alt="Screen Shot 2020-10-31 at 7 58 43 PM" src="https://user-images.githubusercontent.com/1078448/97794169-a538ef80-1bb3-11eb-9000-a15343889f46.png">

Can you indent the code blocks so that they are under the bullets? I find that much easier to follow. 

Alternatively we could move some of these bullets into code comments and not break up into so many pieces (I had only imagined 2-3 code blocks really)

Reply:
I agree, it does look fractured and comments make more sense here than bullets.

Revised as originally discussed: two code snippets (with comments) separated by a brief paragraph on the ``LayoutProvider`` model and plot output to close off the section.

Let me know if you'd like any further changes.